### PR TITLE
docs: update brew install

### PR DESCRIPTION
**Pull request by caarlos0**
refs https://github.com/Homebrew/homebrew-core/pull/136809

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Use a package manager:
 
 ```bash
 # macOS
```
Comment (chenrui333) [inline README.md:59]: would be good to update this reference to `Homebrew (macOS/Linux)`

Comment (maaslalani) [inline README.md:59]:
```suggestion
# macOS or Linux
```

Comment (maaslalani) [reply README.md]: Done!

**Comment by chenrui333** [comment]
awesome 💯

**Pull request merged**